cmd/crdvalidator/handlers/crd.go

```diff
@@ -28,6 +28,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 )
 
+const ValidateAnnotation = "validatecrd"
```

Comment:
could we name this something more kube-like? I know we want to keep the crd as sort of it's own component, but does something generic like `crdvalidation.webhook.io` make sense?

Reply:
Yep, I agree. I think this annotation could also be used for the different levels we are talking about once-upon-a-time so this more generic approach sounds better.